### PR TITLE
1st version for the Numerical matching page

### PR DESCRIPTION
**Pull request by DrLynTaylor**
This is the first version proposal for the Numerical matching page.

### CODE REVIEW

File: match.qmd [hunk 1]
```diff
@@ -2,25 +2,150 @@
...
+-   What is the truth? Which is correct? Can any be considered 'correct'?
 
--   What if SAS and R are equivalent, but a third language yields numerical differences?
+## Historical Background
+
+The statistical analysis system (SAS), was developed by a collective of eight USA southern state universities in the late 1960's. The company SAS institute Inc. was founded in 1976, and their first product release of Base SAS, consisted of approximately 300,000 lines of code^1^. During the first 30 years of existence, SAS software became renowned in the highly regulated medical research industry, for its well documented methodology and its high quality, reproducible, robust and reliable analysis implementation. This made it the number one data analysis tool in the pharmaceutical industry and a gold standard for regulatory submissions around the world.
+
+R has available since 1993 with packages being added continuously, providing a continual development cycle and user-led package development^2^. This open-source language has a package repository called CRAN (Comprehensive R Archive Network) with over 21000 packages in it. The momentum and adoption of R in Pharma is growing, likely due to:
```
Comment (MichaelRimler) [inline match.qmd:17]: 'has been available'?
Add link to CRAN?

File: match.qmd [hunk 2]
```diff
@@ -2,25 +2,150 @@
...
+
+1)  the wide range of additional functionality compared to SAS, which can lead to better efficiency,
```
Comment (MichaelRimler) [inline match.qmd:19]: This is an opinion - are there any data to support? Do we want to position this as R vs SAS - or maybe, this is 'merits of SAS' and 'merits of R'

File: match.qmd [hunk 3]
```diff
@@ -2,25 +2,150 @@
...
+
+If results did differ by a clinically important amount, then it would be very important to further investigate and justify the differences.
+
+### In Practice !
+
+Despite the above, in practice, the medical research industry is governed by strict processes and Standard operative procedures (SOPs) to ensure quality of statistical analysis. For this reason, many companies apply a double programming approach to ensure 100% independent replication of results. This requires a full identical match to be obtained when two programmers do the same work independently. In addition, if the initial work is done by a Contract Research Organization (CRO), working with a pharmaceutical company, the analysis may be programmed a third time to replicate results in the two different companies systems. Finally, when results are submitted to the regulators, they will also program the results and attempt replication.
```
Comment (MichaelRimler) [inline match.qmd:42]: Capitalize Standard Operating Procedures?

File: match.qmd [hunk 4]
```diff
@@ -2,25 +2,150 @@
...
--   What if we the same inputs yield drastically different results?
+-   What if the same inputs yield drastically different results?
 
--   What is the truth? Which is correct?
+-   What is the truth? Which is correct? Can any be considered 'correct'?
 
--   What if SAS and R are equivalent, but a third language yields numerical differences?
+## Historical Background
+
+The statistical analysis system (SAS), was developed by a collective of eight USA southern state universities in the late 1960's. The company SAS institute Inc. was founded in 1976, and their first product release of Base SAS, consisted of approximately 300,000 lines of code^1^. During the first 30 years of existence, SAS software became renowned in the highly regulated medical research industry, for its well documented methodology and its high quality, reproducible, robust and reliable analysis implementation. This made it the number one data analysis tool in the pharmaceutical industry and a gold standard for regulatory submissions around the world.
```
[the remark this answers is not in the document]
Comment (DrLynTaylor) [reply match.qmd:15]: thanks, will update

File: match.qmd (hunk 1 above)
Comment (DrLynTaylor) [reply match.qmd:17]: thanks will update

File: match.qmd (hunk 2 above)
Comment (DrLynTaylor) [reply match.qmd:19]: Good point, definitely not wanting this as SAS vs R!  I'll reword

File: match.qmd [hunk 5]
```diff
@@ -2,25 +2,150 @@
...
+
+R has available since 1993 with packages being added continuously, providing a continual development cycle and user-led package development^2^. This open-source language has a package repository called CRAN (Comprehensive R Archive Network) with over 21000 packages in it. The momentum and adoption of R in Pharma is growing, likely due to:
+
+1)  the wide range of additional functionality compared to SAS, which can lead to better efficiency,
+2)  the large quantity of open source development of packages, which is often conducted through github repositories. In recent times, the [pharmaverse](https://pharmaverse.org/e2eclinical/)^3^ project demonstrates multiple packages designed specifically to resolve the needs of conducting analysis in medical research with a focus on the regulatory needs.
```
[the remark this answers is not in the document]
Comment (DrLynTaylor) [reply match.qmd:20]: Thanks, updated.

File: match.qmd [hunk 6]
```diff
@@ -2,25 +2,150 @@
...
+
+With R being developed in the 1990's, it didn't have such a restriction on speed of computation. This means that the methods that R defaults to, are often the ones that were most commonly used when that package was developed, or that were documented in the literature with better performance compared to an older methodology.
+
+In conclusion, if you write code in SAS and R without specifying fully your analysis using the optional parameters or knowing your default options, it is very likely that the analysis they conduct could be different. In many cases, adding detail to your code (Specifying all options clearly), specifies exactly the analysis you want SAS or R to do, which then ensures that SAS and R are applying the same methods and you get the same results. However, there are still cases where some analyses are only available in SAS and some only in R.
```
[the remark this answers is not in the document]
Comment (DrLynTaylor) [reply match.qmd:26]: Thanks, have added: "It is commonly found that if the user does not change the default options in SAS and R for a particular analysis, then the results that are output are different! But why is this?"    above this section

File: match.qmd (hunk 3 above)
Comment (DrLynTaylor) [reply match.qmd:42]: done. thanks

**Comment by DrLynTaylor** [comment]
I will pull it in, then ask CAMIS team to review and provide feedback/ suggestions. Thanks

**Pull request merged**